src/pages/docs/projects/project-triggers/git-triggers.md

```diff
@@ -0,0 +1,74 @@
+
+Enter a name and description for your trigger.
+
+## Channels and lifecycles
+
+:::div{.hint}
+Git repository triggers create releases based on the default branch in version controlled projects
+:::
+
+If your project contains multiple [channels](/docs/releases/channels), you have the option of selecting which channel this trigger will apply to. The releases created by the trigger will use this channel.
+
+The versions used for those releases is guided by [release versioning](/docs/releases/release-versioning) under **Settings**. They will use the rules defined there.
+
+A preview of the [lifecycle](/docs/releases/lifecycles) used by the selected channel is displayed. You can modify the [lifecycle's phases](/docs/releases/lifecycles/#Lifecycles-LifecyclePhases) to have a release created and deployed to selected environments whenever a new commit is pushed to the monitored repository.
+
+## Trigger sources
+
+Git repositories referenced in your project's deployment process can be selected to be monitored by the trigger to create releases.
+
+Please note that for [configuration as code](/docs/projects/version-control/config-as-code-reference) projects, only steps that reference Git repositories in the deployment process from the **default branch** are able to be referenced. Any changes to the deployment process in other branches will not be available for use in git triggers.
+
+:::figure
+![Repository selection](/docs/projects/project-triggers/images/git-triggers/git-triggers-repository-selection.png)
+:::
+
+### File path filters
+
+When selecting a repository to monitor you will be provided with the option to add file path filters. These filters allow you to specify file paths to include or exclude from the monitoring of new commits.
+
+:::figure
+![File path filters](/docs/projects/project-triggers/images/git-triggers/git-triggers-file-path-filters.png)
+:::
+
+- If no file path filters are specified, all commits to the monitored repository will trigger the creation of a new release. 
+- If file paths are set to be included, only changes to those file paths will be monitored, all other file paths will be excluded.
+- If file paths are set to be excluded, changes to those file paths will not be monitored, all other file paths will be included.
+
+The file path filters support glob patterns and can include the following wildcard characters:
+
+| **Character** | **Description**                           | **Example**                                                                                                           |
+|---------------|-------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|
+| `*`           | Matches multiple characters except `/`    | File path pattern of `release/*` will match the file path `release/1.0.0` but not `release/1.0.0/hotfix1`             |
```

Comment:
These examples look like they were taken from the Git protection rules and look more like branches, it might be good to make them look a bit more like file paths one might see in a repo?

Reply:
Updated